tools/hmac/payload2.json

```diff
@@ -0,0 +1,61 @@
+{
+  "data": {
+    "balancePlatform": "YOUR_BALANCE_PLATFORM",
```

Comment:
Rename file to `account_holder_created_platform_webhook_payload.json`

Reply:
It is an example, it can be any event